src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -915,5 +921,24 @@ private static int GetDeclarationDepth(SyntaxNode? node)
 
             return 0;
         }
+
+        public override SyntaxNode? VisitInterpolatedStringExpression(InterpolatedStringExpressionSyntax node)
+        {
+            if (node.StringStartToken.Kind() == SyntaxKind.InterpolatedStringStartToken)
```

Comment:
i woudl doc this.  specifically the diff between verbatim and non-verbatim.

Reply:
How would you doc this?

Reply:
Explain that newlines are find for verbatim string interpolations but not for non-verbatim ones, and that's why you're setting this flag so that later on we'll suppress adding newlines.